src/js/api/config.js

```diff
@@ -1,7 +1,6 @@
-import _ from 'utils/underscore';
 import { loadFrom, getScriptPath } from 'utils/playerutils';
 import { serialize } from 'utils/parser';
-import { _isValidNumber } from 'utils/underscore';
+import { _isValidNumber, _isNumber, _pick } from 'utils/underscore';
```

Comment:
Remove underscore prefix from these method names so that they will be exported with the correct name.

`import * as _ from 'utils/underscore'` should work the same as `import _` does now.